ps_checkout.php

```diff
@@ -197,22 +170,8 @@ public function install()
     public function installHooks()
     {
         $result = (bool) $this->registerHook(self::HOOK_LIST);
-        /** @var \PrestaShop\Module\PrestashopCheckout\ShopContext $shopContext */
-        $shopContext = $this->getService(\PrestaShop\Module\PrestashopCheckout\ShopContext::class);
 
-        // Install specific to prestashop 1.6
-        if (!$shopContext->isShop17()) {
-            $result = $result && $this->registerHook(self::HOOK_LIST_16);
-            $this->updatePosition(\Hook::getIdByName('payment'), false, 1);
-
-            return $result;
-        }
-
-        // Install specific to prestashop 1.7
-        if ($shopContext->isShop17()) {
-            $result = $result && (bool) $this->registerHook(self::HOOK_LIST_17);
-            $this->updatePosition(\Hook::getIdByName('paymentOptions'), false, 1);
-        }
+        $this->updatePosition(\Hook::getIdByName('paymentOptions'), false, 1);
```

Comment:
```suggestion
        $this->updatePosition(Hook::getIdByName('paymentOptions'), false, 1);
```

you are not using namespaces here, right?